.env

```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=GOPAL
```

Comment:
You can also set your environment variables in netlify .

Reply:
in gitignore add '.env' as it a necessity to hide your environmental variables